tests_e2e/orchestrator/scripts/install-agent

```diff
@@ -115,7 +115,7 @@ echo "Agent's conf path: $waagent_conf_path"
 sed -i 's/AutoUpdate.Enabled=n/AutoUpdate.Enabled=y/g' "$waagent_conf_path"
 # By default GAUpdates flag set to True, so that agent go through update logic to look for new agents.
 # But in e2e tests this flag needs to be off in test version 9.9.9.9 to stop the agent updates, so that our scenarios run on 9.9.9.9.
-sed -i '$a GAUpdates.Enabled=n' "$waagent_conf_path"
+sed -i '$a Debug.EnableGAUpdates=n' "$waagent_conf_path"
```

Comment:
Same update needs in agent update scenario. In this PR, I skip it since I rename the file where this needs to go in another PR.  So it will create a merge conflict and want to avoid that. I'll do it in sperate PR.